apps/studio/components/interfaces/Database/Roles/CreateRolePanel.tsx

```diff
@@ -71,7 +71,7 @@ const CreateRolePanel = ({ visible, onClose }: CreateRolePanelProps) => {
 
   const handleClose = () => {
     onClose()
-    form.reset()
+    form.reset(initialValues)
```

Comment:
this isn't necessary i think

```suggestion
    form.reset()
```

Reply:
sorry! was wrong here - reverting the change